<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt WebBrowser</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="unnamed">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="Frame3">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <layout class="QVBoxLayout" name="unnamed">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="WebAxWidget" name="WebBrowser">
         <property name="control" stdset="0">
          <string>{8856F961-340A-11D0-A96B-00C04FD705A2}</string>
         </property>
         <property name="focusPolicy" stdset="0">
          <enum>Qt::StrongFocus</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="tbNavigate">
   <property name="windowTitle">
    <string>Navigation</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBack"/>
   <addaction name="actionForward"/>
   <addaction name="actionStop"/>
   <addaction name="actionRefresh"/>
   <addaction name="actionHome"/>
   <addaction name="separator"/>
   <addaction name="actionSearch"/>
  </widget>
  <widget class="QToolBar" name="tbAddress">
   <property name="windowTitle">
    <string>Address</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionGo"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>697</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="PopupMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="FileNewGroup_2">
     <property name="title">
      <string>New</string>
     </property>
     <addaction name="actionNewWindow"/>
    </widget>
    <addaction name="actionNewWindow"/>
    <addaction name="FileNewGroup_2"/>
    <addaction name="separator"/>
    <addaction name="actionFileClose"/>
   </widget>
   <widget class="QMenu" name="HistoryMenu">
    <property name="title">
     <string>Hi&amp;story</string>
    </property>
   </widget>
   <widget class="QMenu" name="unnamed">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="PopupMenu"/>
   <addaction name="HistoryMenu"/>
   <addaction name="unnamed"/>
  </widget>
  <action name="actionGo">
   <property name="icon">
    <iconset>
     <normaloff>../go.jpg</normaloff>../go.jpg</iconset>
   </property>
   <property name="iconText">
    <string>Go</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="icon">
    <iconset>
     <normaloff>image1</normaloff>image1</iconset>
   </property>
   <property name="iconText">
    <string>Back</string>
   </property>
   <property name="shortcut">
    <string>Backspace</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="icon">
    <iconset>
     <normaloff>image2</normaloff>image2</iconset>
   </property>
   <property name="iconText">
    <string>Forward</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset>
     <normaloff>image3</normaloff>image3</iconset>
   </property>
   <property name="iconText">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset>
     <normaloff>image4</normaloff>image4</iconset>
   </property>
   <property name="iconText">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="icon">
    <iconset>
     <normaloff>image5</normaloff>image5</iconset>
   </property>
   <property name="iconText">
    <string>Home</string>
   </property>
  </action>
  <action name="actionFileClose">
   <property name="text">
    <string>C&amp;lose</string>
   </property>
   <property name="iconText">
    <string>Close</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset>
     <normaloff>image6</normaloff>image6</iconset>
   </property>
   <property name="iconText">
    <string>Search</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="iconText">
    <string>About</string>
   </property>
  </action>
  <action name="actionAddBookmark">
   <property name="text">
    <string>Add Bookmark</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="iconText">
    <string>About Qt</string>
   </property>
  </action>
  <actiongroup name="FileNewGroup">
   <action name="actionNewWindow">
    <property name="iconText">
     <string>Window</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+N</string>
    </property>
   </action>
  </actiongroup>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QAxWidget</class>
   <extends>QWidget</extends>
   <header>qaxwidget.h</header>
  </customwidget>
  <customwidget>
   <class>WebAxWidget</class>
   <extends>QAxWidget</extends>
   <header location="global">webaxwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
